### ISSUE TITLE
rename `dashboard` to `hub`?

### ISSUE THREAD

**Issue by IgnacioHeredia** [ai4os/ai4-dashboard#79]
`https://dashboard.cloud.ai4eosc.eu`  --> `https://hub.cloud.ai4eosc.eu`

Renaming should have a redirect to be transparent to users.

Why can't we use `https://hub.ai4eosc.eu` instad? Looks cleaner.

**Comment by MartaOB** [comment]
I agree, I believe it's confusing to have a dashboard section inside the dashboard itself. 

I'm not sure about what steps to take in order to make this change correctly.

**Comment by IgnacioHeredia** [comment]
Hi @MartaOB,

@andresheca can guide you (as he manages the current domains).

Steps:

* create a new domains that map all our current domains.
eg. `hub.cloud...` for `dashboard.cloud...` 

* update the reverse proxy to add new domains and add temporary redirects (so the change is transparent to users)
eg. `dashboard.cloud...` --> `hub.cloud...`

* replace dashboard domains in PAPI CORS

As this is quite some work (we have a lot of dashboard related domains), check with @aloga to get the green light to proceed.
If we proceed I would test first with a single domain to be sure that redirects are not breaking anything (eg. retrieving EGI tokens or something simiilar).
